manifests/install.pp

```diff
@@ -187,7 +208,7 @@
     timeout => 1800,
   }
   # Install gitlab with the appropriate package manager (rpm or dpkg)
-  package { 'gitlab':
+  package { "$::gitlab::install::gitlab_pkg":
```

Comment:
keep the `'gitlab'` title, and set this variable as `name`. this way you should have a lot less changes.